doc/10.-The-adsys-daemon.md

```diff
@@ -117,6 +117,11 @@ sssd:
   config: /etc/sssd.conf
   cache_dir: /var/lib/sss/db
 
+# Winbind configuration
```

Comment:
check the comment on the config too. (I start to think we should automate this update too)